boosters/node-beginner/tests/02_exercise.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const app = require("../src/app");
+const userSetup = require("./config/setup")("Users");
+const repoSetup = require("./config/setup")("Repos");
+const teardown = require("./config/teardown");
+const uuid = require("uuid/v4");
+
+const users = [
+  { firstName: "Andrew", lastName: "Mayer", id: uuid() },
```

Comment:
Same name may cause confusion when debugging.

Reply:
Yeah, agreed. The mock data will change in the end but as I was coding I just put in random stuff